src/file-4.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:30781/organizations/org6/rules#rule_key=xoo%3AHasTag))

[See it in SonarQube](http://127.0.0.1:30781/project/issues?id=sample&branch=feature5&issues=AWEzIKgA6L-8XZbwDiyW&open=AWEzIKgA6L-8XZbwDiyW)

[ISSUE_KEY]: AWEzIKgA6L-8XZbwDiyW